.github/workflows/run-daily-rv-gc-binutils.yaml

```diff
@@ -0,0 +1,56 @@
+name: Run-Daily-Binutils-rv-gc
+
+# rv{64/32}gc variants
+
+on:
+  schedule:
+    # Run at 5:00am
+    - cron: 0 5 * * *
+  workflow_dispatch:
+    branches:
+      - main
+
+jobs:
+  # init-submodules sees "binutils_" and lies to us, giving us the binutils hash instead of the gcc hash.
+  # This is intended since it would be a heavy lift to make everything aware of multiple hashes.
+  # The prefix should be enough for us to always know it's a binutils hash
+  init-submodules:
+    uses: ./.github/workflows/init-submodules.yaml
+    with:
+      # Add new prefixes to /dashboard/getdata.py in download_summaries()
+      # ^ Ignored for now since we're testing binutils
+      prefix: 'binutils_'
+      gcchash: '84c87d1f43091c2e537182d029db9739de518096' # Arbitrary fixed GCC hash
```

Comment:
I think we should probably use the gcc-14 release hash `cd0059a1976303638cea95f216de129334fc04d1` since gcc-15 has a lot of things that were re-introduced that are broken like the combine patch

Reply:
```suggestion
      gcchash: 'cd0059a1976303638cea95f216de129334fc04d1' # Arbitrary fixed GCC hash
```